packages/inter-protocol/src/vaultFactory/storeUtils.js

```diff
@@ -109,7 +121,79 @@ const fromVaultKey = key => {
   return [decodeNumber(numberPart), vaultIdPart];
 };
 
+/**
+ * Sort key by ratio in descending debt.
+ *
+ * @param {NormalizedDebt} normalizedDebt
+ * @param {Amount<'nat'>} collateral
+ * @returns {string} lexically sortable string in which highest
+ * debt-to-collateral is earliest
+ */
+const toCollateralizationRatioKey = (normalizedDebt, collateral) => {
```

Comment:
this is the same as `toVaultKey` but without the vaultId differentiator.  I don't see it used anywhere except tests. What's the purpose?